static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx

```diff
@@ -111,43 +112,34 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
 
   const docKeysMap = useMemo(() => makeDocKeyMap(currentPlatform?.id), [currentPlatform]);
 
-  const isPlatformSupported = useMemo(() => {
-    if (!currentPlatform) {
-      return false;
-    }
-    return supportedProfilingPlatforms.includes(
-      // typescript being typescript
-      currentPlatform.id as SupportedProfilingPlatform
-    );
-  }, [currentPlatform]);
-
   const {docContents, isLoading, hasOnboardingContents} = useOnboardingDocs({
     docKeys: docKeysMap ? Object.values(docKeysMap) : [],
     project: currentProject,
-    isPlatformSupported,
+    isPlatformSupported: true,
```

Comment:
If a platform is unsupported, we would've aborted already in the wrapper, so we need to rework this a little to be able to show the unsupported message.